kde/qtcurve/Portfile

```diff
@@ -1,21 +1,26 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
-# $Id$
-# kde/libkgapi/Portfile
+# kate: backspace-indents true; indent-pasted-text true; indent-width 4; keep-extra-spaces true; remove-trailing-spaces modified; replace-tabs true; replace-tabs-save true; syntax Tcl/Tk; tab-indents true; tab-width 4;
```

Comment:
I would have preferred if support for kate was addressed globally rather than just on a single Portfile.

Not that I ever figured out how support for vim was supposed to work anyway (it works for me as the last line of the file, but not in this configuration), but nonetheless I see a limited value of having special support in/for just a single file in the whole repository.